### PR TITLE
static page

### PR DESCRIPTION
**Pull request by nminhduc**

### CODE REVIEW

**Comment by thanhtt** [comment]
ok

File: app/assets/javascripts/static_pages.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (dieunb) [inline app/assets/javascripts/static_pages.coffee]: remove this file

File: app/assets/stylesheets/static_pages.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the StaticPages controller here.
```
Comment (dieunb) [inline app/assets/stylesheets/static_pages.scss]: remove this file

File: app/helpers/static_pages_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module StaticPagesHelper
```
Comment (dieunb) [inline app/helpers/static_pages_helper.rb]: remove this file

File: app/assets/stylesheets/custom.css.scss [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+}
+
+.center {
+  text-align: center;
+}
+
+.center h1 {
+  margin-bottom: 10px;
+}
+
```
Comment (thanhtt) [inline app/assets/stylesheets/custom.css.scss]: nested lại ông

File: app/assets/stylesheets/custom.css.scss [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+footer small {
+  float: left;
+}
+
+footer ul {
+  float: right;
+  list-style: none;
+}
+
```
Comment (thanhtt) [inline app/assets/stylesheets/custom.css.scss]: nested ô

**Pull request merged**